modules/developer-edition/version-control.php

```diff
@@ -61,7 +61,7 @@ public static function on_activate_and_deactivate_plugin() {
 	 * @return object
 	 */
 	public function pre_set_site_transient_update_plugins( $transient ) {
-		$current_version = $this->get_elementor_plugin_data()['Version'];
+		$current_version = $this->get_plugin_version( Bootstrap::ELEMENTOR_PLUGIN_NAME );
```

Comment:
Adapt to work with the pro plugin. (add a filter to `get_wp_org_data` in order to allow the pro plugin to set its  stable version)